### PR TITLE
Re-fluff plutonium cells and adjust balance of atomic items

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Re-fluff plutonium cells into plutonium fuel cells; adjust consumer atomic items"```

#### Purpose of change
Plutonium cells cause a bit of a lore problem, as their game function is to represent "sci fi" power storage items, with huge power outputs over short times. Plutonium is a terrible descriptive choice for that, since unless these are coke-can-sized nuclear reactors, any nuclear battery has a *small* power output over a *long* time. What's worse, plutonium cells are used all over the place in weird ways that are sometimes making them into long term batteries, other times draining them quickly to power a plasma gun.

This has been causing issues with unclear development direction, as contributors think that plutonium cells should be used to make all kinds of micro nuclear things.

#### Describe the solution
- Plutonium cells are renamed to Plutonium Fuel Cells. This is a lateral shift and the only purpose is to make the name imply that they're not just a battery powered by plutonium.
- Rewrote plutonium fuel cell description: `This is neither a fuel cell, nor nuclear, but the name stuck.  It uses plutonium-244 as a catalyst to stabilize a complicated nanocompound that can store enormous amounts of power.  Unfortunately it cannot be recharged by conventional means: expended cells had to be sent to a central reprocessing facility that almost certainly doesn't exist anymore.`
- Changed atomic lamp and nightlight to use betavoltaic cells. Added betavoltaic cells. Changed "nightlight" to "reading light" to make purpose to player more clear, but added a reference to using them as nightlights.
- Changed atomic coffeepot to use a curium powered radioisotope thermoelectric generator, which while more realistic is if anything perhaps *more* insane than before, if only because it's less vague so we can see just how insane it really is. The new atomic coffeepot isn't hot enough to be a hotplate. I would love to see it get the ability to thaw frozen food without heating / cooking.
- Now that atomic lamps/lights aren't vague proprietary rivtech things, I made them much easier to craft. If you have the battery it's really not hard.

#### Describe alternatives you've considered
There's a fair argument that the lamp and nightlight could be using capacitors to store power when not in use. I see no reason to code that in explicitly though, since I haven't added a clear wattage to the BV cells. It's probably happening behind the scenes.

As mentioned above, I would love to see the coffeepot be able to thaw items, just not heat or cook them. I do not have the skill to add that.

I may add the possibility of finding betavoltaic cells in electronics item groups, if they're used in lamps and lights they're probably not *that* rare. RTGs could also spawn naturally but that's a little crazier.

#### Additional context
Microreactors need a larger scale rework, which is still in the planning phase. For now it's probably okay for them to continue using plutonium fuel cells as a power source.

I would like plutonium cells to drop a "depleted" version of themselves when expended, but I don't know how to go about adding that in.

I'm hoping that now that the black box of "rivtech sci fi" is removed, people will start doing crazy stuff with betavoltaics and RTGs salvaged from these consumer-electronics-with-no-regard-to-efficiency-nor-public-safety

### CODE REVIEW

**Comment by FrozenSnowFox** [comment]
The bionic power CBMs give plutonium cells when disassembled.  Considering the description states they can't be recharged shouldn't this be changed to some other power source as well?  It doesn't really make sense to use a non rechargeable power source in the unit when you use various bionics to recharge it.

**Comment by I-am-Erk** [comment]
Yes, probably. We'll have to see what would be a reasonable alternative.

File: data/json/items/tools.json [hunk 1]
```diff
@@ -257,26 +257,25 @@
...
     "weight": 4102,
-    "volume": 3,
+    "volume": "1000 mL",
     "price": 100000,
     "to_hit": -2,
     "bashing": 5,
     "material": [ "plastic", "aluminum" ],
     "symbol": ",",
     "color": "light_green",
-    "qualities": [ [ "BOIL", 1 ] ],
```
Comment (Rail-Runner) [inline data/json/items/tools.json]: Is removing BOIL quality intended? It's vital for this item to be able to make coffee without any external container.

Comment (I-am-Erk) [reply data/json/items/tools.json]: Right, I forgot that it wasn't a use effect. I'll put that back.

**Comment by TechyBen** [comment]
Nice to see some real thought going into this!

IRL less than power station sized nuclear power generally scales similar to normal batteries. A betavoltaic cell/chip gives out similar power and lifespan to say a lithium button cell of similar size. You get different power output curves/lifespans to some extent, depending on if the device has constant draw (great for the betavoltaic) or pulsed/punctuated draw events (batteries good for this).

The use case is kinda strange. But for a reading light these kinda exist (keyrings with tiny amounts of reactive materials that then have luminescent paint stuff so it glows). So the night light is doable IRL to some extent. More so if a capacitor was used for night only use.

Also, betavoltaics are rather easy to craft if the components are available (we have plutonium slurry in the game right? XD ).
https://hackaday.com/2016/12/01/make-your-own-nuclear-battery/

**Comment by I-am-Erk** [comment]
I question those energy density results a little, or at least assume they're missing some context, given that the battery in your linked example is about the size of a couple AA batteries once fully wrapped, and puts out less than 2 microwatts.

Regardless, I left the energy output of our betavoltaics intentionally vague. I'd guess it's somewhere in the high μW to low mW range, which is why I have four of them powering a low light lamp.

**Comment by Human-Shield** [comment]
This is defiantly needed (along with CBM power being better explained). Devices that use betavoltaic should have a special brand name like "Forever" or even what we have with "Atomic".

**Comment by RarkGrames** [comment]
The refugee center merchant also wants plutonium from you during one of his quests.

**Comment by I-am-Erk** [comment]
I'll have to check their dialogue.  The actual quest is probably fine.

**Comment by I-am-Erk** [comment]
Ah, yep, thanks for the heads up. That mission's gonna be a problem, I'll probably rewrite it.

**Comment by I-am-Erk** [comment]
There. That's a temporary fix both for the mission and for the faction... They can't actually make RTGs from the plutonium in a fuel cell, it's plut244. They'll need a new mission later to get a more permanent power source. Maybe securing an advanced hydrogen electrolysis tool from a lab and getting some fuel cells, then they could sell you hydrogen.

**Comment by Mister-Cultist** [comment]
now there's an idea! The merchants being able to sell hydrogen after enough missions are done would be a great way to give access to the item without relying on blind luck since it's such a useful item and you can barely find any of it.

**Comment by TechyBen** [comment]
> I question those energy density results a little, or at least assume they're missing some context, given that the battery in your linked example is about the size of a couple AA batteries once fully wrapped, and puts out less than 2 microwatts.
Ah sorry. There are two. The current DIY/industry standard and the theoretical/lab proven but no production model "perfect result" with a diamond wafer design (so think our progress from old Solar panels to now, but if we'd been in the CDDA universe so "atomic energy" had followed with the tech too).

So it means you can choose anything between the two and meet the laws of physics, just cost/time would scale accordingly. But we already assume "cheap home atomic power" exists in CDDA so it's down to the story/gameplay as to what we choose. :)

**Comment by RarkGrames** [comment]
Found an other instance of plutonium, the description of the reactor core extension device says that the tool can be used to "double the amount of plutonium it can hold".

**Comment by I-am-Erk** [comment]
The minireactor's time on this earth is limited. Not too worried about that.

**Comment by RarkGrames** [comment]
This item seems to double the magazine size of the plutonium cell operated tools, not the minireactor.

**Comment by I-am-Erk** [comment]
Ah yeah, apparently so. I'll have to get it in another PR

**Pull request merged**